libavutil/hwcontext_d3d12va.h

```diff
@@ -124,6 +124,11 @@ typedef struct AVD3D12VAFrame {
  *
  */
 typedef struct AVD3D12VAFramesContext {
+    /**
+     * This field is used for resource creation.
+     */
+    D3D12_RESOURCE_FLAGS Flags;
+
```

Comment:
Please add the new field member to the end of the structure.

Reply:
Done.